CONTRIBUTING.md

```diff
@@ -6,9 +6,19 @@ Thanks a lot for thinking about contributing to Firenvim! Please do not hesitate
 
 ### Requirements
 
-Installing from source requires NodeJS, npm, and Neovim >= 0.4.
+Installing from source without docker requires NodeJS, npm, and Neovim >= 0.4.
 
-### Cross-browser steps
+Installing from source using docker requires docker 18.09 or higher for [BuildKit support](https://docs.docker.com/develop/develop-images/build_enhancements/). Older Docker versions will build the required files into the image, but will not copy them into the host.
```

Comment:
Maybe it would be better to have the requirements be part of the build instructions? I think it might be clearer. Let me know what you think :)

Reply:
Can you please clarify? do you mean to put the requirements for each build setup (docker/non-docker) near the build instructions?

Reply:
>Can you please clarify? do you mean to put the requirements for each build setup (docker/non-docker) near the build instructions?

Yes, exactly, sorry I wasn't clear enough :)

Reply:
Done